### DISCUSSION TITLE
Zoto Star Sky Projector with Laser and WB3S Chip

### DISCUSSION THREAD

**Discussion by kxn25** [arendst/Tasmota#18688] [Support request]
Hello,
i want to flash Tasmota on the WB3S and need the configuration of the pin´s.
Or i want to replace it with a EPS-12S or 8266? and also need the proper config.
I have tryed with ESP-12S with Tasmota flashed befor solder in! But the ESP12 is not booting up?! so i cant even get to configure the pins.
On the WB3S are all pins used exept TXD2.

This did not help me at all. and also does not have the Laser controlled.
[https://templates.blakadder.com/led_star_projector.html](url)

Here sime Pictures of the Projector.
![photo1684404837 (1)](https://github.com/arendst/Tasmota/assets/44605046/0fbf25a4-cf32-43d1-b130-929fe6665fc3)
![photo1684404837](https://github.com/arendst/Tasmota/assets/44605046/94611822-5faa-4879-93f4-cb10f66f6f04)

Please some one Help me on this. i dont like Clouds in my House :-)

**Comment by sfromis** [comment]
If you flashed correctly, the ESP-12S module should be able to boot with the same wiring as used for successful flashing, except for not grounding gpio0, of course. Running a serial terminal with baud rate 115200, you should get a bunch of Tasmota boot messages, or the chip boot messages before that on 74880.

**Comment by kxn25** [comment]
Tasmota is running Just fine before i solderd it. 
Flashed with latest tasmota.
And is running after aswell.
While solderd in on rx tx ist no data comming.
It seams that the boot up configuration is diffrend to the wb3S!

**Reply by sfromis** [reply]
Well, if it is *specifically* [ESP-12S](https://tasmota.github.io/docs/Pinouts/#esp-12s) (and not some other ESP-12 variant), the strapping pins should be handled internally in the module. I'd triple-check the soldering carefully. Maybe try if it helps grounding gpio 15.

**Comment by kxn25** [comment]
![photo1684481640](https://github.com/arendst/Tasmota/assets/44605046/e7f66142-a8c0-45fe-963f-75a3830443e0)


i will try with GPIO15 to ground.
